widgetssdk/src/main/java/com/glia/widgets/config/UiComponents.java

```diff
@@ -0,0 +1,75 @@
+package com.glia.widgets.config;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.util.TypedValue;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.glia.widgets.config.components.RemoteColor;
+import com.glia.widgets.config.components.RemoteConfiguration;
+import com.glia.widgets.config.components.base.Alignment;
+import com.glia.widgets.config.components.base.DpSize;
+import com.glia.widgets.config.components.base.SpSize;
+import com.glia.widgets.config.components.base.TextStyle;
+import com.glia.widgets.config.deserializers.AlignmentDeserializer;
+import com.glia.widgets.config.deserializers.DpDeserializer;
+import com.glia.widgets.config.deserializers.RemoteColorDeserializer;
+import com.glia.widgets.config.deserializers.SpDeserializer;
+import com.glia.widgets.config.deserializers.TextStyleDeserializer;
+import com.google.android.material.shape.CornerFamily;
+import com.google.android.material.shape.MaterialShapeDrawable;
+import com.google.android.material.shape.ShapeAppearanceModel;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class UiComponents {
+    private static float applyDimension(int unit, float value, Context context) {
+        return TypedValue.applyDimension(unit, value, context.getResources().getDisplayMetrics());
+    }
+
+    public static float dpToPx(float value, Context context) {
```

Comment:
FYI, to avoid having several functions with the same purpose, there is a `convertDpToPixel()` in `ResourceProvider.java`. Ideally, better to leave it only in one place.